benchmarks_test.go

```diff
@@ -0,0 +1,75 @@
+package rocket_test
+
+import (
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/dannypsnl/rocket"
+)
+
+var rk *rocket.Rocket
+
+func TestMain(m *testing.M) {
+	rk = rocket.Ignite(":8080")
```

Comment:
I think the problem is, we don't exactly run a server at the test, and we didn't get real information about request ops.